src/lib/mbtiles_import_worker.js

```diff
@@ -151,11 +151,20 @@ function importMbTiles({
   db.close()
   mbTilesDb.close()
 
-  // Ensure a final progress event is sent (because of throttle)
-  port.postMessage({
-    type: 'progress',
+  const baseFinalMessage = {
     importId,
     soFar: bytesSoFar,
     total: totalBytes,
+  }
+
+  // Ensure a final progress event is sent (because of throttle)
+  port.postMessage({
+    ...baseFinalMessage,
+    type: 'progress',
+  })
+
+  port.postMessage({
+    ...baseFinalMessage,
+    type: 'complete',
```

Comment:
liked the idea of a having an explicit `complete` message that's sent from the worker to indicate that the import finished. open to removing though